src/component/container/element-list.tsx

```diff
@@ -128,38 +136,76 @@ export class ElementList extends React.Component {
 				store.execute(ElementLocationCommand.addChild(element, draggedElement));
 				store.setSelectedElement(draggedElement);
 			},
-			children: items,
-			active: element === selectedElement
+			children: [...slots, ...defaultSlotItems],
+			active: element === selectedElement && !store.getSelectedSlot()
 		};
 	}
 
-	public createItemFromProperty(
-		key: string,
-		value: PropertyValue,
+	public createItemFromSlot(
+		slot: Slot,
+		element: PageElement,
 		selectedElement?: PageElement
 	): ListItemProps {
-		if (value instanceof Array) {
-			const items: ListItemProps[] = [];
-			(value as (string | number)[]).forEach((child, index: number) => {
-				items.push(this.createItemFromProperty(String(index + 1), child));
-			});
-			return { value: key, children: items };
-		}
+		const store = Store.getInstance();
+		const slotId = slot.getId();
+		const children: PageElement[] = element.getContent(slotId);
+		const childItems: ListItemProps[] = [];
+		const selectedSlot = store.getSelectedSlot();
 
-		if (value === undefined || value === null || typeof value !== 'object') {
-			return { label: key, value: String(value) };
-		}
+		children.forEach((value: PageElement, index: number) => {
+			childItems.push(
+				this.createItemFromElement(
+					children.length > 1 ? `Child ${index + 1}` : 'Child',
+					value,
+					selectedElement
+				)
+			);
+		});
 
-		if (value instanceof PageElement) {
-			return this.createItemFromElement(key, value, selectedElement);
-		} else {
-			const items: ListItemProps[] = [];
-			Object.keys(value).forEach((childKey: string) => {
-				// tslint:disable-next-line:no-any
-				items.push(this.createItemFromProperty(childKey, (value as any)[childKey]));
-			});
-			return { value: key, children: items };
-		}
+		const updateSelectedSlot: React.MouseEventHandler<HTMLElement> = event => {
+			event.stopPropagation();
+			store.setSelectedElement(element);
+			store.setSelectedSlot(slotId);
+			store.setElementFocussed(false);
+		};
+
+		const slotListItem: ListItemProps = {
+			value: `🔘 ${slot.getName()}`,
```

Comment:
Can we change this Unicode character into an escape sequence? I know editors that have issues with unusual characters.

Reply:
i'll see what i can do :)